R/sass.R

```diff
@@ -40,13 +40,13 @@
 #'   sass_file(tmp_file)
 #' ))
 sass <- function(input = NULL, options = sass_options(), output = NULL,
-  cache_options = sass_cache_options(), write_attachments = NA) {
+  write_attachments = NA, cache = sass_cache_get()) {
```

Comment:
How come the order here has changed?

Reply:
The `cache` param is not the same as the previous `cache_options`, so we're already effectively changing the order (by removing the old option and inserting a new one). This would break any existing code that used unnamed args up to those positions. I figured that since we're making that potential breaking change anyway, this would be a good time to revisit the order. `cache` seems to the parameter that's least likely to be used, so I moved it to the last position.

Reply:
Good point, thanks.